packages/ts-fdc3-oms/src/components/Portfolio/Portfolio.tsx

```diff
@@ -161,29 +161,26 @@ const Portfolio = (): JSX.Element => {
         }
     }
 
-    const raiseOrderIntent = (
+    const raiseOrderIntent = useCallback((
         side: '1' | '2',
         instrument: Fdc3Instrument,
         quantity?: number
     ) => {
-        glue.intents
-            .raise({
-                intent: 'NewOrder',
-                context: {
-                    data: {
-                        order: {
-                            type: 'fdc3.order',
-                            side: side,
-                            instrument: instrument,
-                            quantity: quantity,
-                            notes: selectedFund?.name,
-                        },
-                    },
-                },
-                target: 'reuse',
-            })
+        if (window.fdc3) {
+            window.fdc3
+                .raiseIntent('NewOrder', {
+                    type: 'fdc3.order',
+                    order: {
+                        type: 'fdc3.order',
+                        side: side,
+                        instrument: instrument,
+                        quantity: quantity,
+                        notes: selectedFund?.name
+                    }
+                })
```

Comment:
you must add the correct "target" - if it is reause in glue.intents - it means that the target is the current Portfolio app instance, or another open app (like TradeHistory) for example, but we gotta put the correct one. Syntax is { appId: "/app id in json condfig definition/"}

Reply:
in this case i would bet it targets the Orders app, but you need to double check